### PR TITLE
docs/rewrite vignettes

### PR DESCRIPTION
**Pull request by sidohaakma**
Checkout the branch and run:

```
install.packages("pkgdown")
pkgdown::build_site()
```

### CODE REVIEW

**Comment by sidohaakma** [comment]
[Adding Angela's script to the vignettes](https://trello.com/c/1nqv07mH/398-adding-angelas-script-to-the-vignettes)

**Comment by fdlk** [comment]
Shall we move the table data to the molgenis-r-armadillo repository?

**Comment by sidohaakma** [comment]
[Update vignettes on DSMolgenisArmadillo](https://trello.com/c/h6cDIk8v/410-update-vignettes-on-dsmolgenisarmadillo)

**Comment by sidohaakma** [comment]
> Shall we move the table data to the molgenis-r-armadillo repository?

Done!

File: vignettes/DSMolgenisArmadillo.Rmd.orig [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+               password = "",
+               token = token,
+               table = "gecko/2_1-core-1_0/nonrep",
+               driver = "ArmadilloDriver")
+
+# create loginframe
+logindata <- builder$build()
+logindata
+
+datashield.logout(conns)
```
Comment (fdlk) [inline vignettes/DSMolgenisArmadillo.Rmd.orig:93]: Quitting from lines 72-97 (vignettes/DSMolgenisArmadillo.Rmd.orig) 
Error in datashield.logout(conns) : object 'conns' not found

**Review by fdlk — APPROVED** [review]

**Pull request merged**